docs/admin/kubeadm.md

```diff
@@ -820,6 +820,15 @@ This means you can, for example, prepopulate `/etc/kubernetes/pki/ca.crt`
 and `/etc/kubernetes/pki/ca.key` with an existing CA, which then will be used
 for signing the rest of the certs.
 
+## Self-hosting the Kubernetes control plane  {#self-hosting}
+As of 1.8, kubeadm can experimentally create a _self-hosted_ Kubernetes control
+plane. This means that key components such as the API server, controller
+manager, and scheduler run as normal pods instead of static pods.
+
+Self-hosting is beta in kubeadm 1.8 and is expected to become the default in
+kubeadm 1.9. To create a self-hosted cluster, pass the
+`--feature-gates=SelfHosting=true` to `kubeadm init`.
```

Comment:
Tell the user about the Limitations (don't reboot :smile:)
Can also tell a bit more about the actual process described in https://github.com/kubernetes/kubeadm/blob/master/docs/design/design_v1.8.md

Reply:
Done. Please let me know if what I wrote is technically accurate.